python/pyspark/sql/session.py

```diff
@@ -2197,13 +2197,15 @@ def copyFromLocalToFs(self, local_path: str, dest_path: str) -> None:
             messageParameters={"feature": "SparkSession.copyFromLocalToFs"},
         )
 
-    @remote_only
     def interruptAll(self) -> List[str]:
```

Comment:
can we fix docs too?

Reply:
`python/docs/source/reference/pyspark.sql/spark_session.rst`

Reply:
Updated docs. Thanks!